pydocumentdb/synchronized_request.py

```diff
@@ -134,9 +134,6 @@ def _Request(connection_policy, requests_session, resource_url, request_options,
         return (response.raw, headers)
 
     data = response.content
-    if not six.PY2:
```

Comment:
`.decode()` is unnecessary here &mdash;

![image](https://user-images.githubusercontent.com/6472374/43184469-61a932b6-8ff1-11e8-85eb-5cea108664f7.png)